class-tgm-plugin-activation.php

```diff
@@ -457,11 +450,11 @@ public function add_plugin_action_link_filters() {
 		 * @since 2.5.0
 		 *
 		 * @param array $actions Action links.
-		 *
```

Comment:
I seem to remember that we both said we preferred to have the line between @param's and @return lines, so why not do so ?
This is not WP Core after all (yet) and the empty line between @param and @return also complies with all the examples included in the upcoming PSR-5.